### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by Adewunmi97**
- The project involves the use of  HTML, CSS, and bootstrap.
-Added webhint and stylehint.
-Fixed changes required by TSE's.

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,370 @@
...
+}
+
+.lh-2 {
+  line-height: 1.2em;
+}
+
+.divider {
+  height: 1px;
+
+  /* background-color: rgba(0,0,0,0.1); */
```
Comment (MuriloRoque) [inline styles.css]: Please, remove this commented code, I think you missed it :wink:

**Comment by Adewunmi97** [comment]
-All webhint errors fixed

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
# Status: Changes requested :red_circle: 

### Hello @Adewunmi97 and @oracleot :wave: 

### Your page is looking so good, it's almost identical to the original, I can hardly notice the differences. We only have some little details to fix:

## Page

### Navigation bar

***Medium Screen***

- [x] Your logo should be smaller, just like the original, check below:

**YOURS:**

Screenshot from 2020-08-27 13-43-45
<br>
**ORIGINAL:**

Screenshot from 2020-08-27 13-43-48
<br>

### Main Content

***Medium Screen***

- [x] Your 'Free Signup Section' width is too big compared to the original, check below:

**YOURS:**

Screenshot from 2020-08-27 13-48-05
<br>

**ORIGINAL:**

Screenshot from 2020-08-27 13-48-08
<br>

***Small Screen***

- [x] I can't see the title of the first section, you need to add some margin, look below:

Screenshot from 2020-08-27 13-53-01

<br>

- [x] I can't find the 'More Stories' section in your page, check the original:

Screenshot from 2020-08-27 13-57-40

<br>

- [x] Now here's something that most students miss: **'The Famous Newsletter Section'**, the image vanishes on small screens :fearful:  Check below:

**YOURS:**

Screenshot from 2020-08-27 14-00-50
<br>
**ORIGINAL:**

Screenshot from 2020-08-27 14-00-54
<br>

## General

- [ ] Just check my single comment in your code below :wink: 

## Summary

- Your code is beautiful, everything is in its place, all good HTML and CSS practices :+1: 
- You mastered the use of Bootstrap, it's really awesome, congratulations :clap: :clap:
- Great repository, very professional, keep up the good work :rocket:  
- We still need to fix some details about the layout, but most of it is in place :smile: 

## Kindly make the requested changes and submit for another review, please :pray: 

## I will be on Slack 24/7 in case you have any questions about this review :handshake: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Review by bertil291utn — CHANGES_REQUESTED** [review]
# **Status:** Changes requested 🟠 
<p align="center"> image <br><strong>Not yet</strong> </p>

# Hey guys, you've been doing a great job, and also I hope you enjoyed coding.

### Your work is amazing you master it GitHub Flow you have a professional README file. 

However, in order to accomplish as the actual page, you required to do these changes, listed below:

### General requirements 
- [ ] Remove `.vscode` directory or add to .gitginore file
- [ ] Reove extra padding in the page
image


### Specific req.

| Section/Screen                             | Large | Medium | Small |
| :----------------------------------------- | :---: | :----: | :---: |
| Navbar              |      |      |       |
| Featured stories              |      |        |       |
| Top story             |       |        |       |
| My turn             |       |        |   1    |
| Cultural&Travel             |      |        |    1   |
| Opinion                 |       |        |       |
| Latest news                  |       |        |       |
| More stories |      |        |       |
| In the magazine         |       |        |       |
| Editor's pick |    |     |      |
| Footer                                     |       |        |       |
###### This table is given each section with his screen width approach, each issue is given by number below this table

1.  After more stories section you're missing to add my turn and culture& travel sections
image

### Summary

-  Remove extra padding entire page
- You're missing two sections for small screens 

***

Guys work in the requested changes above and request another review.
Good luck. 

> “Never stop, never conform until what is good becomes even better, and what’s better becomes excellent.”

**Review by SpaYco — APPROVED** [review]
# Project Approved

thank you for making the required changes, your project is now approved!

### please don't forget to merge to master, good luck with the next projects!


Best, 
Aziz Mejri 
_contact me on [twitter](https://twitter.com/iSpaYco) or slack (@SpaYco) if you need any help_

**Pull request merged**